src/goto-instrument/contracts/contracts.cpp

```diff
@@ -887,11 +978,12 @@ void code_contractst::check_frame_conditions(
   const irep_idt &function,
   goto_programt &body,
   goto_programt::targett &instruction_it,
+  const goto_programt::targett &instruction_end,
```

Comment:
See earlier commit: pass this by value (there isn't much of a point of passing an iterator by reference).

Reply:
Done; thanks!